Microsoft.Toolkit.Uwp.UI.Media/Enums/ImageBlendMode.cs

```diff
@@ -55,6 +57,5 @@ public enum ImageBlendMode
         Luminosity = 23,
         Subtract = 24,
         Division = 25,
-        #pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Isn't this still needed at the end of the file?

Reply:
Why? We're doing the same in the `HighPerformance` package as well (in `Box<T>`).
If there are no other things in the rest of the file isn't it fine?

Reply:
Didn't know that, awesome! Thanks!